src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Colors[] allColors = Colors.values();
+        Random random = new Random();

```

Comment:
```suggestion
public class ColorSupplier {
    private Random random = new Random();

    public String getRandomColor() {
		Colors[] allColors = Colors.values();
```
це поле краще до класу перенести так як воно не змінюється і нам не потрібно створювати новий об'єкт для кожного виклику методу